backend/src/main/java/com/woowacourse/moamoa/review/service/ReviewService.java

```diff
@@ -37,4 +39,30 @@ public Long writeReview(final Long githubId, final Long studyId, final WriteRevi
         final Review review = Review.writeNewReview(study.getId(), member.getId(), writeReviewRequest.getContent());
         return reviewRepository.save(review).getId();
     }
+
+    public void updateReview(final Long githubId, final Long reviewId, final EditingReviewRequest editingReviewRequest) {
+        final Member member = memberRepository.findByGithubId(githubId)
+                .orElseThrow(MemberNotFoundException::new);
+        final Review review = reviewRepository.findById(reviewId)
+                .orElseThrow(ReviewNotFoundException::new);
+
+        if (!review.isReviewer(member.getId())) {
+            throw new UnwrittenReviewException();
+        }
+
+        review.updateContent(editingReviewRequest.getContent());
```

Comment:
```suggestion
        review.updateContent(new Reviewer(member.getId()), editingReviewRequest.getContent());
```
위와 같이 하고 updateContent 안에서 예외를 던지도록 하는게 좋아 보입니다.
그 이유는 updateContent 안에 넣어주면 domain 단위 테스트에서 예외가 던져지는 케이스를 확인할 수 있어요.
controller나 service 테스트에서는 NotFound 케이스와 전체적인 성공 케이스(얘는 인수에서 봐도 될 듯 합니다.) 하나만 봐도 될 꺼 같아요.

Reply:
이렇게 바꾸니까 Review에서 단위 테스트로 충분히 가능하네요!